lib/experimental/fonts/font-library/font-library.php

```diff
@@ -56,10 +56,18 @@ function wp_register_font_collection( $config ) {
 	}
 }
 
-// @core-merge: This code needs to be removed.
 add_action(
 	'enqueue_block_editor_assets',
 	function () {
 		wp_add_inline_script( 'wp-block-editor', 'window.__experimentalFontLibrary = true', 'before' );
 	}
 );
+
+$default_font_collection = array(
+	'id'             => 'google-fonts',
+	'name'           => 'Google Fonts',
+	'description'    => __( 'Add from Google Fonts. Fonts are copied to and serverd, from your server.' ),
+	'data_json_file' => path_join( __DIR__, 'google-fonts.json' ),
```

Comment:
> google-fonts.json file ideally will be removed when https://github.com/WordPress/gutenberg/issues/53992 is merged.

It would be great to avoid maintaining the JSON file in the repository that is over 50 kB.